packages/utilities/src/focus.ts

```diff
@@ -56,17 +56,19 @@ export function getLastFocusable(
  * @param currentElement - The descendant of rootElement to start the search at.  This element is the first one checked,
  * and iteration continues forward.  Typical use passes rootElement.firstChild.
  * @param includeElementsInFocusZones - true if traversal should go into FocusZone descendants.
+ * @param checkNode - Include currentElement in search when true. Defaults to true.
  * @public
  */
 export function getFirstTabbable(
   rootElement: HTMLElement,
   currentElement: HTMLElement,
-  includeElementsInFocusZones?: boolean
+  includeElementsInFocusZones?: boolean,
+  checkNode: boolean = true
```

Comment:
It would be nice to prevent this function from suffering a similar fate of `getNextElement` and having a difficult-to-parse list of booleans pile up in the parameter list.

Since you always call this with `false`, is there a semantically meaningful name that could be given to a new function that does what you want?

Reply:
I don't think so. This is really a specialized version of `getNextElement` and `checkNode` has the same contextual meaning for both.

Reply:
I don't disagree with the comments regarding API surface but rather than refactor one method I'm going for consistency now. I think the API could be refactored as a whole as a separate effort, though.